docs/virtual_machines/startup_scripts.md

```diff
@@ -112,6 +113,17 @@ ConfigDrive data source:
         cloudInitConfigDrive:
           userData: "#cloud-config"
 
+When using the ConfigDrive datasource, the `networkData` part has to be in the
+[OpenStack Metadata Service Network](https://specs.openstack.org/openstack/nova-specs/specs/liberty/implemented/metadata-service-network-info.html) format:
```

Comment:
... or in ENI (legacy) format, see https://cloudinit.readthedocs.io/en/latest/reference/network-config-format-eni.html#network-config-eni.

Reply:
I don't think we should encourage using a deprecated format which we haven't tested.

Reply:
Yes, we don't want to encourage it, but it is supported. IMHO I don't see the OpenStack format being desirable either.

Reply:
> Yes, we don't want to encourage it, but it is supported. 
Supported by cloud-init does not  necessarily mean supported by KubeVirt.

>  IMHO I don't see the OpenStack format being desirable either.
The reason for supporting this format is fact that credentials API works with `CloudInitConfigDrive` (at least for now).

Reply:
> The documentation about this is quite unclear. Can you rework this part too? E.g. what is supported and under which circumstances? https://kubevirt.io/user-guide/virtual_machines/startup_scripts/#network-config

I agree that this part should be reworked.
I can have a look at this part, but it will require more research.
I think it is out of the scope of this PR.